client/go/outline/config/config_iptable.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2025 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/netip"
+
+	"github.com/Jigsaw-Code/outline-apps/client/go/configyaml"
+	"github.com/Jigsaw-Code/outline-apps/client/go/outline/iptable"
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+)
+
+type ipTableRootConfig struct {
+	Table    []ipTableEntryConfig  `yaml:"table"`
+	Fallback configyaml.ConfigNode `yaml:"fallback,omitempty"`
+}
+
+type ipTableEntryConfig struct {
+	IP     string                `yaml:"ip"`
```

Comment:
I just realized something: the IP must replaces with a list of IPs, because we want to be able to list many prefixes and specify the dialer only once.

For example, these are the [Google IP prefixes](https://www.gstatic.com/ipranges/goog.json). We need to be able to use the same dialer for each of them. Not doing so adds overhead in the config and memory usage, and can hurt in other ways (like separating caching that a dialer may do).

The config should look like:

```yaml
table:
  - ips:
       - 8.8.4.0/24
       - 8.8.8.0/24
       - 8.34.208.0/20
       ...
    dialer: {name: dialerA}
fallback: name: default
```